keras_nlp/models/bloom/bloom_mlp.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_nlp.backend import keras
+from keras_nlp.backend import ops
+
+
+class BloomMLP(keras.layers.Layer):
+    def __init__(
+        self,
+        hidden_dim,
+        dropout=0.0,
+        kernel_initializer="glorot_uniform",
+        bias_initializer="zeros",
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.hidden_dim = hidden_dim
+        self.dropout = dropout
+        self.kernel_initializer = kernel_initializer
+        self.bias_initializer = bias_initializer
+
+    def _gelu_function(self, x):
```

Comment:
We have two gelu versions we can use. `keras.activations.gelu`, and `keras.activations.gelu(x, approximate=True)`. Will neither work here?

Reply:
`keras.activations.gelu(x, approximate=True)` will work fine actually I have tried it now. The reason I used that implementation is to match the [huggingface implementation](https://github.com/huggingface/transformers/blob/c48787f347bd604f656c2cfff730e029c8f8c1fe/src/transformers/models/bloom/modeling_bloom.py#L120) . I thought this implementation will result in different result compared to `keras.activations.gelu(x, approximate=True)`.